src/RoadRegistry.Product.Projections/RoadRegistry.Product.Projections.csproj

```diff
@@ -0,0 +1,17 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="..\..\packages\Be.Vlaanderen.Basisregisters.Build.Pipeline\Content\Be.Vlaanderen.Basisregisters.Build.Pipeline.Settings.Library.props" />
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <RootNamespace>RoadRegistry.Product.Projections</RootNamespace>
+    <AssemblyName>RoadRegistry.Product.Projections</AssemblyName>
+    <Nullable>disable</Nullable>
+  </PropertyGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\RoadRegistry.Product.Schema\RoadRegistry.Product.Schema.csproj" />
+    <ProjectReference Include="..\RoadRegistry.BackOffice\RoadRegistry.BackOffice.csproj" />
```

Comment:
@yreynhout I'm guessing we'll keep this reference to `BackOffice.csproj`, since we'll be using it eg here: https://github.com/Informatievlaanderen/road-registry/pull/20/files#diff-615bfa8e0c379d0a1bbad44d0404fb8aR30

Reply:
Yes, not only for that type, but all the events are defined in that project as well, so no point in trying to _lose the reference_.